doc/tutorials/dockerfile.rst

```diff
@@ -148,19 +150,19 @@ For a Dockerfile to work on Binder, it must meet the following requirements:
 
    .. code-block:: Dockerfile
 
-       # Make sure the contents of our repo are in ${HOME}
-       COPY . ${HOME}
-       USER root
-       RUN chown -R ${NB_UID} ${HOME}
-       USER ${NB_USER}
+      # Make sure the contents of our repo are in ${HOME}
+      COPY . ${HOME}
+      USER root
+      RUN chown -R ${NB_UID} ${HOME}
+      USER ${NB_USER}
 
    This chown is required because Docker will be default
    set the owner to ``root``, which would prevent users from editing files. Note that the repository 
    should in general be clone with ``COPY``; although ``RUN git clone ...`` is a valid command for the
    ``Dockerfile``, it does not invalidate the build cache of mybinder. Thus, if available, the the cached
    repository will be used even after changes to the repository. 
 
-5. It must accept command arguments. The Dockerfile will effectively be launched as:
+5. It must accept command arguments.
```

Comment:
It was incorrect to say it would be launched like this. What's missing is captured in point 3 and the debugging section I'm adding about not allowing use of `root`.